packages/devtools_shared/lib/src/extensions/extension_manager.dart

```diff
@@ -77,7 +77,7 @@ class ExtensionsManager {
 
       for (final extension in extensions) {
         final config = extension.config;
-        if (config is! Map) {
+        if (config is Map) {
```

Comment:
I don't think we want this  - this means we are now failing when the config is a map (instead of the original intention to fail when the config is a map).

Reply:
+1 this introduces a bug. Please change back to `is!`.